store.go

```diff
@@ -30,7 +30,7 @@ type Store interface {
 	New(r *http.Request, name string) (*Session, error)
 
 	// Save should persist session to the underlying store implementation.
-	Save(r *http.Request, w http.ResponseWriter, s *Session) error
+	Save(w http.ResponseWriter, s *Session) error
```

Comment:
This changes the public API [breaking change]. It would break all existing store implementations, of which there are many.